frontend/src/app/organization/organization-contact/organization-contact.component.ts

```diff
@@ -0,0 +1,30 @@
+import {Component, EventEmitter, Input, OnInit, Output, ViewChild} from '@angular/core';
+import { NgForm } from '@angular/forms';
+import { OrganizationContactPayload } from '../organization.model';
+
+@Component({
+  selector: 'volontulo-organization-contact',
+  templateUrl: './organization-contact.component.html',
+  styleUrls: ['./organization-contact.component.css']
+})
+export class OrganizationContactComponent implements OnInit {
+  @ViewChild('contactForm') contactForm: NgForm;
+  @Output() contact = new EventEmitter<OrganizationContactPayload>();
+  @Input() contactStatus: Number;
+
+  constructor() { }
+
+  ngOnInit() {
+  }
+  onSubmit() {
+    this.contact.emit({
+      name: this.contactForm.value.name,
+      email: this.contactForm.value.email,
+      phone_no: this.contactForm.value.phone_no,
+      message: this.contactForm.value.message,
+    });
+    if (this.contactStatus === 201) {
```

Comment:
Does it work? I might be wrong, but it seems when this condition is checked the API call (as it's kind of "fire and forget") has not yet finished.